CMakeLists.txt

```diff
@@ -11,18 +25,16 @@ project( ACEtk LANGUAGES CXX )
 set( CMAKE_CXX_STANDARD 17 )
 set( CMAKE_CXX_STANDARD_REQUIRED YES )
 
-option( ACEtk.tests "Build the ACEtk unit tests" OFF )
-option( ACEtk.python "Build ACEtk python bindings" ON )
-option( strict_compile
-    "Treat all warnings as errors." ON
-    )
-
-# Compile flags
-set( common_flags "-Wall" "-Wextra" "-Wpedantic" )
-set( strict_flags "-Werror" )
```

Comment:
As a note, these two still appear as `${PREFIX}_X_flags` in `src/legacy/interpretation`. I think they were already empty strings so it's no big deal, and `CMAKE_RELEASE_FLAGS` and similar do most of what is needed.

Reply:
Once we update to use shacl cmake, we'll add the flags back in. For now, I just want to make sre that all njoy libraries use the same thing everywhere.